### PR TITLE
Convert Ember Data Models to native classes

### PR DESCRIPTION
**Pull request by Windvis**
This converts all models that use the classic class syntax to native classes.

### CODE REVIEW

File: app/models/bestuursorgaan-classificatie-code.js [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
   // This is what mu-cl-resources uses to search on, and how the model will be presented while editing relationships.
-  stringRep: collect.apply(this,['id', 'label', 'scopeNote']),
-
-  label: attr(),
-  scopeNote: attr(),
-  standaardType: hasMany('bestuursfunctie-code', { inverse: 'standaardTypeVan' }),
-  isClassificatieVan: hasMany('bestuursorgaan', { inverse: null}),
-  uri: attr()
-});
+  get stringRep() {
```
Comment (Windvis) [inline app/models/bestuursorgaan-classificatie-code.js:12]: I couldn't find any references in the codebase to this property. Is this still needed?

File: app/models/zitting.js [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
+  @hasMany('behandeling-van-agendapunt', { inverse: null }) behandeldeAgendapunten;
+  @belongsTo('editor-document', { inverse: null }) notulen;
+  @hasMany('agenda', { inverse: null }) publicatieAgendas;
+  @hasMany('agendapunt', { inverse: 'zitting' }) agendapunten;
+  @belongsTo('functionaris', { inverse: null }) secretaris;
+  @belongsTo('mandataris', { inverse: null }) voorzitter;
+  @hasMany('mandataris', { inverse: null }) aanwezigenBijStart;
+  @hasMany('mandataris', { inverse: null }) afwezigenBijStart;
+
+  get isPublished() {
```
Comment (Windvis) [inline app/models/zitting.js:20]: I don't think this is used anywhere?

File: app/models/kandidatenlijst.js [hunk 3]
```diff
@@ -1,16 +1,17 @@
...
+  @attr lijstnummer;
+  @belongsTo('rechtstreekse-verkiezing', { inverse: 'heeftLijst' }) rechtstreekseVerkiezing;
+  @hasMany('persoon', { inverse: 'isKandidaatVoor' }) kandidaten;
+
+  rdfaBindings = {
     lijstnaam: "http://www.w3.org/2004/02/skos/core#prefLabel",
     lijstnummer: "http://data.vlaanderen.be/ns/mandaat#lijstnummer",
     kandidaten: "http://data.vlaanderen.be/ns/mandaat#heeftKandidaat",
     rechtstreekseVerkiezing: "http://data.vlaanderen.be/ns/mandaat#behoortTot",
     "lijsttype": "http://data.vlaanderen.be/ns/mandaat#lijsttype"
```
Comment (Windvis) [inline app/models/kandidatenlijst.js:15]: The model doesn't have an attribute / relationship named like this, can this be removed?

File: app/models/bestuursorgaan-classificatie-code.js (hunk 1 above)
Comment (nvdk) [reply app/models/bestuursorgaan-classificatie-code.js:12]: I believe this used to be used in an auto generated admin interface, where this probably originated. should be safe to remove now.

File: app/models/kandidatenlijst.js (hunk 3 above)
Comment (nvdk) [reply app/models/kandidatenlijst.js:15]: it seems like it does exist in the backend: https://github.com/lblod/app-gelinkt-notuleren/blob/f3b160e22bb63b239a728a2477c34411057f8d41/config/resources/slave-mandaat-domain.lisp#L47 so we either add the relationship or remove it from the bindings. both are fine for me

File: app/models/zitting.js (hunk 2 above)
Comment (nvdk) [reply app/models/zitting.js:20]: I think this is related to old code indeed, should be safe to be removed.

**Review by nvdk — APPROVED** [review]
probably needs some testing to make sure, but code wise I don't see any issues,

File: app/models/kandidatenlijst.js (hunk 3 above)
Comment (Windvis) [reply app/models/kandidatenlijst.js]: I removed the binding since the `lijsttype` model doesn't exist yet in the frontend either.

**Pull request merged**